05/styles/desktop.css

```diff
@@ -0,0 +1,92 @@
+section {
+    padding: 10px;
+    margin-bottom: 15px;
+}
+
+/* 
+=============================
+Nav Section
+=============================
+*/
+
+.nav__section .container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.nav__logo { 
+    width: 200px; 
+    height: 80px;
+}
+
+.nav__menu {
+    width: 400px;
+    height: 80px;
+}
+
+/* 
+=============================
+Header Section
+=============================
+*/
+
+.header__section .container {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    padding: 10px 0;
+}
+
+.header__section .content,
+.content--smaller  {
+        width: 49%;
+        height: 350px;
+        background-color: orange;
+}
+
+/*     
+=============================
+Article Section
+=============================
+*/
+
+.article__section .container {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+}
+
```

Comment:
Staraj sie nie zmieniać specyficzności dla selektorów, jeśli potrzebuejsz zmian to użyć modufikatora tj.
```
.container {
    display: flex;
    justify-content: space-between;
    flex-direction: row;
}

.container--column {
    flex-direction: column;
}
```

Wtedt w HTML piszesz: `class="container container--column"`